tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -30,7 +30,7 @@
 <fe_dai.conf>
 <ssp.conf>
 <dmic.conf>
-<hw_config.conf>
+<hw_config_cardinal_clk.conf>
```

Comment:
we probably need to introduce intel prefixes?

Reply:
this file is in the platform/intel folder already, do you still want to add the intel prefix?

Reply:
we should have something like <intel/hw_config_cardinal_clk.conf> then to avoid conflicts. the 'flat' naming and search path guarantees someone will screw up one day.

Reply:
ack fixed now